web-console/.eslintrc.js

```diff
@@ -0,0 +1,167 @@
+module.exports = {
```

Comment:
This file (`.js`) should have the Apache header

Reply:
Done